client/go/build.sh

```diff
@@ -1,7 +1,14 @@
 # Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
 
 # Execute from this directory to build the command-line client to bin/vespa
-export GOPATH=`pwd`
-cd "$GOPATH/src/"
-go test ./...
+cd util
```

Comment:
Just do `go test ./...` to run all tests recursively.

Reply:
zsh$ go test ./...
pattern ./...: directory pkg/mod/github.com/davecgh/go-spew@v1.1.1/spew outside available modules

Reply:
This is likely because you have `pkg` in current directory as a result of using `GOPATH` in the past, and mixing it with the project path. Try remove the `pkg`, make sure GOPATH isn't set and run go test again.

Reply:
A+ support, that worked.